app/src/main/AndroidManifest.xml

```diff
@@ -96,6 +97,21 @@
             android:name=".settings.SettingsActivity"
             android:parentActivityName=".webview.WebViewActivity" />
 
+        <activity
+            android:name=".nfc.NfcActivity"
+            android:label="@string/nfc_title_nfc_processing">
+            <intent-filter>
+                <action android:name="android.nfc.action.NDEF_DISCOVERED" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data
+                    android:scheme="https"
+                    android:host="www.home-assistant.io"
+                    android:pathPrefix="/nfc/" />
```

Comment:
`/tag/`?

Reply:
I added `/tag/` as well. The `/nfc/` is still needed for the universal links with the `url?=`, right?

Reply:
I think that might be up to you - the iOS app uses `/ios/nfc?url=…` for those. I did not include support for `/nfc/?tag=xyz` in the latest app, but we could add it remotely via the url config.

Reply:
Okay, I think I'll use the `/ios/nfc/` url then. Otherwise it would require changes for you too. Plus you can't generate those URLs in the Android app itself.